nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Cli.Test/EntryPointTests.Run.cs

```diff
@@ -58,7 +58,7 @@ await RunAsync(
             );
         }
 
-        private static async Task RunAsync(TestFile[] files, Job job, string[] expectedUrls, MockNuGetPackage[]? packages = null)
+        private static async Task RunAsync(TestFile[] files, Job job, string[] expectedUrls, MockNuGetPackage[]? packages = null, string? repoContentsPath = null)
```

Comment:
This change isn't required for this PR, but it's a change I commonly make so I can easily test real-world jobs.  I'm sick of changing this so I'm adding it.